specification/monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metrics_API.json

```diff
@@ -171,8 +171,8 @@
           "description": "the sum of all of the values in the time range."
         },
         "count": {
-          "type": "integer",
-          "format": "int64",
+          "type": "number",
+          "format": "double",
```

Comment:
It seems like "count" should be a whole number. Can you get a count of 1.5?

Reply:
Yeap, we discuss whether to change it in the backend (and leave this spec as it was) or change it in the spec.  The reason for it to be a double (NOTE: we do not generate the metric, we just pass it on) is uncertain, but it might have something to do with the fact that this could actually be the average of several counts.

We thought that it after that, we decided to change the spec to reflect the current response of the backend to minimize customer impact.

Reply:
OK, you might consider changing the name in the future then, since a count shouldn't be able to be a double AFAIK